src/SayMore/Model/Fields/FieldInstance.cs

```diff
@@ -204,11 +203,29 @@ public static IEnumerable<string> GetMultipleValuesFromText(string text)
 			var list = text.Split(new[] { kDefaultMultiValueDelimiter, kAlternateMultiValueDelimiter },
 				StringSplitOptions.RemoveEmptyEntries);
 
+			list = ListWithoutRoles(list);
+
 			return (from val in list
 					where val.Trim() != string.Empty
 					select val.Trim());
 		}
 
+		/// ------------------------------------------------------------------------------------
+		/// <summary>
+		/// Unique sorted list of names with role removed from name if present
```

Comment:
The comment mentions names, but the method name doesn't. I think I like NamesWithoutRoles better for clarity. Code change is good though.